src/V3Active.cpp

```diff
@@ -237,27 +242,45 @@ class ActiveNamer final : public ActiveBaseVisitor {
 public:
     // METHODS
     AstScope* scopep() { return m_scopep; }
-    AstActive* getCActive(FileLine* fl) {
-        if (!m_cActivep) {
-            m_cActivep = new AstActive(
-                fl, "combo", new AstSenTree(fl, new AstSenItem(fl, AstSenItem::Combo())));
-            m_cActivep->sensesStorep(m_cActivep->sensesp());
-            addActive(m_cActivep);
+
+    // Return an AstActive sensitive to the given special sensitivity class
+    template <typename SenItemKind> AstActive* getSpecialActive(FileLine* fl) {
+        static_assert(std::is_same<SenItemKind, AstSenItem::Static>::value
+                          || std::is_same<SenItemKind, AstSenItem::Initial>::value
+                          || std::is_same<SenItemKind, AstSenItem::Final>::value
+                          || std::is_same<SenItemKind, AstSenItem::Combo>::value,
+                      "Not special sensitivity");
+
```

Comment:
That would be a run-time error (instead of a compile time error), which would be unfortunate. What we want is `if constexpr`, but that's C++17. Anyhow refactored this to avoid the duplication.